src/pages/Club/Club/Club.tsx

```diff
@@ -91,13 +91,17 @@ const Club = () => {
 
   const removeMember = async (followerID: number) => {
     await removeFollower(followerID);
-    await createNotification(activeUserID as string, "На жаль, ви були виключені з прихильників куреня",true);
+    await createNotification(activeUserID as string, "На жаль, ви були виключені з прихильників куреня", true);
     const response = await getClubById(+id);
     setFollowersCount(response.data.followerCount);
     setFollowers(followers.filter((f) => f.id !== followerID));
     setCanJoin(true);
 }
   const addMember = async () => {
```

Comment:
if I'm not mistaken this notification should appear when user was deleted from members of Курінь, but your method still named as `addMember `

Reply:
Your'e right, but also the method has correct name because of what it do. I used that type of notification at them because I need to say the user that he/she was removed from his/her previous club after user become the follower of  the new club.